pkgs/by-name/fc/fcitx5-rose-pine/package.nix

```
@@ -0,0 +1,33 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
```

```suggestion
stdenvNoCC.mkDerivation {
```
passing finalAttrs as an argument is optional, you can omit this part as it isn't used in this package.